synapse/lib/stormlib/gen.py

```diff
@@ -425,6 +453,53 @@ class LibGen(s_stormtypes.Lib):
             [ geo:place=(gen, name, $geoname) :name=$geoname ]
             return($node)
         }
+
+        function fileBytesBySha256(sha256, try=$lib.false) {
+            ($ok, $sha256) = $__maybeCast($try, hash:sha256, $sha256)
+            if (not $ok) { return() }
+
+            file:bytes:sha256=$sha256
```

Comment:
Should we prefer non-guid based file:bytes (e.g. file:bytes=$sha256 then file:bytes:sha256=$sha256)?

Reply:
From what I've seen, if you create a file:bytes node with a sha256, it always sets :sha256. So I think that makes `file:bytes=$sha256` redundant? I can bring it up as a goback tomorrow.

Reply:
Fixed based on today's discussion.